### ISSUE TITLE
watir-webdriver is not supported

### ISSUE THREAD

**Issue by bootstraponline** [applitools/Eyes.Selenium.Ruby#4]
```
Failures:

  1) Applitools visits the marketing page
     Failure/Error: @eyes.open(app_name: app_name, test_name: test_name, driver: @driver)
     Applitools::EyesError:
       Driver is not a Selenium::WebDriver::Driver (Watir::Browser)
     # /Users/user/.rvm/gems/ruby-2.2.2/gems/eyes_selenium-2.13.0/lib/eyes_selenium/eyes/eyes.rb:135:in `open'
     # ./spec/applitools_spec.rb:15:in `visual_test'
     # ./spec/applitools_spec.rb:22:in `block (2 levels) in <top (required)>'
```

**Comment by bootstraponline** [comment]
Here is a patch that should work:

```ruby
elsif driver.is_a?(Appium::Driver) || driver.is_a?(Watir::Browser)
      @driver = Applitools::Driver.new(self, {driver: driver.driver, is_mobile_device: true})
```

**Comment by danielputerman** [comment]
Thanks! It's a good suggestion, and I'll add it to the code.
Generally speaking, the Ruby SDK can integrate with Watir (just like it can integrate with any WebDriver based framework) by simply providing the WebDriver to Eyes when calling *eyes.open*.

**Comment by bootstraponline** [comment]
It works fine with the watir browser.driver workaround, it'd just be a bit cleaner without it. Thanks for responding.

**Comment by danielputerman** [comment]
Done :)

**Comment by bootstraponline** [comment]
Nice :+1:

**Comment by Githraine** [comment]
Was this implemented?
I am having issues with this tool for Watir, and the link to the source code does not include the suggested code to support it...

**Comment by danielputerman** [comment]
Hi, yes, it was implemented and should be working with Watir. Notice that Watir seems to have issues with Ruby 2.4.0 regardless of Applitools.  I just double checked and everything seems to be working correctly. 
If you're not using Ruby 2.4.0, and having specific problem with Applitools, please contact Applitools support and they'll help you:  support [at] applitools dot com

**Comment by danielputerman** [comment]
My mistake, there was a bug in version 2.39 which created a problem with Watir.

**Comment by danielputerman** [comment]
Fixed in fc10e2dfb87e1f498c2ddb85bce46ab15115e000. 

Thanks for the feedback @Githraine !

**Comment by Githraine** [comment]
IIt looks good now, Thank you!
